src/GameLogic/ItemPowerUpFactory.cs

```diff
@@ -184,12 +184,25 @@ private IEnumerable<PowerUpWrapper> GetBasePowerUpWrappers(Item item, AttributeS
         attribute.ThrowNotInitializedProperty(attribute.BaseValueElement is null, nameof(attribute.BaseValueElement));
         attribute.ThrowNotInitializedProperty(attribute.TargetAttribute is null, nameof(attribute.TargetAttribute));
 
-        yield return new PowerUpWrapper(attribute.BaseValueElement, attribute.TargetAttribute, attributeHolder);
-
-        var levelBonus = (attribute.BonusPerLevelTable?.BonusPerLevel ?? Enumerable.Empty<LevelBonus>()).FirstOrDefault(bonus => bonus.Level == item.Level);
-        if (levelBonus is not null)
+        var levelBonusElmt = (attribute.BonusPerLevelTable?.BonusPerLevel ?? Enumerable.Empty<LevelBonus>())
```

Comment:
Are these changes still required?

Reply:
Yes. 187-190 deleted lines went into the 200-205 added lines in the `else` statement.

Reply:
I mean, is the special handling for jewellery still required? Shouldn't it work already without it? 😀

Reply:
The way I see it, yes it is.
Without this special handling, if you equipped, for example, two rings of ice, one +2, one +3, you would have a `CompositeAttribute` with the the elements list: [0.1, 0.2, 0.1, 0.3], thus, when getting the maximum value, you would get 0.3. With this special handling your list now becomes [0.3, 0.4], thus the maximum value is 0.4, which is the correct resistance. 
Do you see a better way?

Reply:
Ah, now I understand. Thank you for the explaination.
I think it could make sense here to apply the new logic for all items, not just jewellery. What do you think?

Reply:
Do you mean always making a `CombinedElement` for the base + level bonus elements whenever there is a level bonus element? Yes, that seems to make sense.

Reply:
So, can you change this, when you have time? Thank you :)

Reply:
@sven-n , PR updated.
For this change the wings' dmg absorption and increase level bonus tables also needed to be updated. There was also a bug here, as the values of level bonus were being compounded instead of added together as a unique multiplier.
For example, with the following wings:
![Screenshot from 2024-10-08 10-22-26](https://github.com/user-attachments/assets/813f08a5-f19e-454b-acb6-f456208c75b8)

**Before changes**:
- for the dmg increase you would have ~81% (1.39*1.3) instead of 69%:
![Screenshot from 2024-10-08 10-35-30](https://github.com/user-attachments/assets/1fbd0d3a-5e92-4229-86e9-9d3282e46cbf)
- for the dmg absorb you would have ~57% (1 - 0.61*0.7) instead of 69%:
![Screenshot from 2024-10-08 10-33-54](https://github.com/user-attachments/assets/8ceb1181-d182-46c4-852a-3f264b6904de)

**After changes,** with a `CombinedElement`:
- now actual 69% increase:
![Screenshot from 2024-10-08 10-20-41](https://github.com/user-attachments/assets/40e001a4-9c53-4851-b2c2-428004e4fb76)
- now actual 69% absorption
![Screenshot from 2024-10-08 10-21-29](https://github.com/user-attachments/assets/0e733634-8c02-49ee-b081-8554c20887b8)